src/opera/util/input_validation.py

```diff
@@ -195,15 +195,18 @@ def check_amp_mean_geometry_burst_ids(cslc_input_burst_ids, ancillary_file_list,
         If the set of ancillary file burst_ids do not match the set of cslc input file burst_ids.
 
     """
-    ancillary_burst_ids = get_burst_id_set(ancillary_file_list, logger, name)
-    # Test that there is a unique burst_id for each burst
+    ancillary_burst_ids = check_disp_s1_ancillary_burst_ids(ancillary_file_list, logger, name)
+    # Test none of the ancillary inputs have the same burst ID
     if len(ancillary_burst_ids) != len(ancillary_file_list):
-        msg = f'Burst ids in {ancillary_file_list} are not unique.'
+        msg = f"Duplicate burst ID's in ancillary file list. " \
+              f"Lenth of file list {ancillary_file_list}, Length of file set {ancillary_burst_ids}"
+        # msg = f'Burst ids in {ancillary_file_list} are not unique.'
         logger.critical(name, ErrorCode.INVALID_INPUT, msg)
-    # Verify that the sets of burst_ids are equal
+    # Verify that the sets of ancillary input burst ID's match the set of CSLC input burst ID's
     if ancillary_burst_ids != cslc_input_burst_ids:
-        msg = f'input_burst_ids: {cslc_input_burst_ids} do not match ancillary_burst_ids: ' \
-              f'{ancillary_burst_ids} for {ancillary_file_list}'
+        msg = f"Set of input CSLC burst IDs do not match the set of ancillary burst IDs: " \
+              f"In cslc set, but not in ancillary set: {cslc_input_burst_ids.difference(ancillary_burst_ids)} " \
+              f"In ancillary set, but not in cslc set: {ancillary_burst_ids.difference(cslc_input_burst_ids)}"
```

Comment:
New message looks good, but as is it would print as one long line which might be hard to read. Let's add some "\n" line breaks to the end of each string so the formatting matches the way you've broken up the string here.